### ISSUE TITLE
Nitrogen supply bug

### ISSUE THREAD

**Issue by DerFlammenwerfer** [Sierracode/-tg-station#122]
Checked supply controls and the pumps. All were operational as they are supposed to be, but see pic related for the bug.
![bug](https://cloud.githubusercontent.com/assets/22332543/19013419/10579b76-879e-11e6-9d53-1f57c7f334a1.png)

The nitrogen refuses to pump after I gave it a volumetric upgrade.

**Comment by AssBlastProductions** [comment]
This has been a problem for a long time, Since late Pizza days, it's really awful when it happens to a supermatter engine you're trying to build. Assuming this is the same issue another symptom of this happening is that the pipe section after the pump seemed to not exist at all, it was clearly visible and there but if you tried to unwrench it nothing would happen as if you had just tried to wrench the floor but at the same time if you attempted to place down a new section of pipe it would tell you that there was already a section of pipe on that tile.

It seems to be prone to happening with volume pumps being fitted onto pipes where both sides already have gas in them and when you apply pipes to floor tiles rather than plating. Not sure what might be causing it but it's most likely something that goes way back since I distinctly remember this happening all the way back during Pizza. It might have to do with that one atmos update we had ages ago that made gases move through pipes in real time or whatever it was that made the gas in bombmix making update while canisters were still wrenched in instead of the canisters only starting to be affected after unwrenching them.

In short it looks like an old problem that's been around for a while where sections of pipe will sometimes for some reason behave as though they don't exist but at the same time do exist and prevent new pipes from being placed.

**Comment by Sierracode** [comment]
Known /tg/ bug cleaning issue tab blah blah blah
